index.html

```diff
@@ -5007,13 +4980,254 @@ <h3 id="looping-AudioBufferSourceNode">
             times (in seconds) must be converted to the appropriate
             sample-frame positions in the buffer according to this sample-rate.
           </p>
+        </section>
+        <section>
+          <h3 id="playback-AudioBufferSourceNode">
+            Playback of AudioBuffer Contents
+          </h3>
+          <p>
+            This section specifies how the playback of the contents of the
+            buffer is to be handled, accounting for the fact that playback is
+            influenced by the following factors working in combination, which
+            can vary dynamically during playback:
+          </p>
+          <ul>
+            <li>A starting offset, which can be expressed in terms of fractions
+            of a sample.
+            </li>
+            <li>Loop points, which can be expressed in terms of fractions of a
+            sample.
+            </li>
+            <li>Playback rate and detuning parameters, which combine to yield a
+            single computed playback rate that can assume non-infinite values
+            which may be positive or negative.
+            </li>
+          </ul>
           <p>
-            When scheduling the beginning and the end of playback using the
-            <code>start()</code> and <code>stop()</code> methods, the resulting
-            start or stop time MUST be rounded to the nearest sample-frame in
-            the sample rate of the <a><code>AudioContext</code></a>. That is,
-            no sub-sample scheduling is possible.
+            The algorithm to be followed internally to generate output from an
+            <a>AudioBufferSourceNode</a> conforms to the following principles:
           </p>
+          <ul>
+            <li>Resampling of the buffer may be performed arbitrarily by the UA
+            at any desired point to increase the efficiency or quality of the
+            output.
+            </li>
+            <li>Sub-sample start offsets or loop points may require additional
```

Comment:
I don't think this really needs to be specified.  The algorithm below basically implies some kind of interpolation because the playback head is not guaranteed to be on a sample boundary.